api/src/types/session.ts

```diff
@@ -0,0 +1,25 @@
+import { User } from '@peterportal/types';
+import 'express-session';
+
+declare module 'express-session' {
+  export interface SessionData {
+    /**
+     * Store custom data in passport
+     */
+    passport: PassportData;
+    /**
+     * URL to return to when finish authentication
+     */
+    returnTo: string;
+  }
+
+  export interface PassportData {
+    /**
+     * True if is validated as an admin
+     */
```

Comment:
great, but is it necessary?

Reply:
the comment? probably not. I just cut/pasted this from another file.